cranelift/filetests/filetests/isa/x86/windows_fastcall_x64_unwind.clif

```diff
@@ -153,13 +153,13 @@ block0(v0: i64, v1: i64):
 ; sameln: UnwindInfo {
 ; nextln:     version: 1,
 ; nextln:     flags: 0,
-; nextln:     prologue_size: 26,
+; nextln:     prologue_size: 25,
```

Comment:
The astute reader may ask, "why did prologue sizes change if this was supposed to be a no-op cleanup-style change?"

It turns out my `F64 causes a whole XMM register to be preserved anyway.` comment became a lie between when I wrote it and when #1216 landed. Or maybe I was wrong the whole time? It was generating `movsd` (for example, `f2 41 0f 11 33`), where with this change Cranelift generates `movups` (-> `41 0f 11 33`, one byte shorter).

This is why prologues are shorter, but more importantly, `movsd` only works with the low 64 bits of its xmm argument! `movups` is the correct instruction (`movaps` would also be acceptable), which moves the entire 128 bits.

Reply:
Aha! I was about to ask that very question. (Don't know how astute of a reader I am, though.) Thanks for the explanation.